kong/llm/proxy/handler.lua

```diff
@@ -477,4 +484,57 @@ function _M:access(conf)
 
 end
 
+
+
+function _M:init_worker_for_plugin(plugin_name)
```

Comment:
a narrowed name would be better.

Reply:
would be good to check if any listener has enabled http2, and avoid patch it if none has enabled http2 to avoid performance degration as possible

Reply:
We do not know which configuration has been configured after Kong started i think (I mean nginx conf, because http2 enable or disable was configured by kong config -> nginx conf)

Reply:
done